subsys/bluetooth/controller/hci_driver.c

```diff
@@ -490,6 +490,14 @@ static int configure_supported_features(void)
 		if (err) {
 			return -ENOTSUP;
 		}
+
+		sdc_hci_cmd_vs_periodic_adv_event_length_set_t params = {
+			.event_length_us = CONFIG_BT_CTLR_SDC_PERIODIC_ADV_EVENT_LEN_DEFAULT
```

Comment:
Do we have to change something inside the lib to ensure this value is used after HCI reset?

Reply:
That's a good point